index.js

```diff
@@ -0,0 +1,16 @@
+function popUpMenu() {
+  document.querySelector('#mobile-container').classList.toggle('hide-mobile-content');
+  document.querySelector('#mobile-menu').classList.toggle('show-mobile-menu');
+  //  document.querySelector('header').classList.toggle()
+}
+
+function removeMenu() {
+  document.querySelector('#mobile-container').classList.toggle('hide-mobile-menu');
+  document.querySelector('#mobile-menu').classList.toggle('show-mobile-menu');
+  //  ['mobile-backgrond-image-1','mobile-backgrond-image-2'].map( className =>
+  //  document.querySelector('#image2').classList.toggle(className));
+}
```

Comment:
You have some commented lines of code in lines 4, 10, & 11. It is recommended that you always delete commented lines/blocks of code. If you need to keep track of such lines, you can do so with Git history.